debug.xml

```diff
@@ -320,6 +323,10 @@
         DELETE FROM notifications WHERE id = :id;
     </entry>
 
+    <entry key='database.clearPositionsHistory'>
+        DELETE FROM positions WHERE deviceid = :deviceId and servertime &lt; :serverTime;
```

Comment:
What if the last position was a month ago and you delete everything older than a week?